docs/components/datepicker/index.md

```diff
@@ -0,0 +1,30 @@
+<script setup>
+import React from './react.md';
+import data from './data.json';
+import { mapFrameworkStatuses } from '../utils.js';
+</script>
+
+# Date Picker - BETA
+
+This component is still in a beta phase, and you might encounter bugs when using it. 
+If you encounter anything, please report it to the WARP team: [#smp-warp-design-system](https://sch-chat.slack.com/archives/C04P0GYTHPV)
+
```

Comment:
Missing the component status component:

<components-status v-bind="mapFrameworkStatuses(data.frameworks)" />